packages/python-packages/api-stub-generator/apistubgen.py

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
This file seems like a duplicate of the console entry point already defined in the package?

Reply:
API tool runs parser as script as opposed to console application( apistubgen). Let me check if I can reuse the code in console entry point.

Reply:
@annatisch Fixed and reused console code in apistub module